### PR TITLE
eslintrc fix and update readme

### PR DESCRIPTION
**Pull request by whilei**

### CODE REVIEW

File: src/lib/rpcApi.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import 'isomorphic-fetch';
 
-const url = 'http://localhost:1920';
```
Comment (dulanov) [inline src/lib/rpcApi.js]: 1920 is a default port for emerald connector

Comment (sjmackenzie) [inline src/lib/rpcApi.js]: get off my lawn!

Comment (whilei) [inline src/lib/rpcApi.js]: Ahah, got it. Figured there must be a reason.

Comment (whilei) [inline src/lib/rpcApi.js]: Change is reverted in d0d1a18.

**Pull request merged**